ModelDerivative_01/instructions/task-2.md

```diff
@@ -50,21 +50,21 @@ In this tutorial, you will use a Postman environment variable named `ossBucketKe
    
    You should see a screen similar to the following image:
    
-   ![Signed url](../images/task2_obtain_signed_url_new.png "Signed url")
+   ![Signed url](../images/t1_task2_obtain_signed_url_new.png "Signed url")
    
 ## Upload the file
 
 1. Download the file *box.ipt* from the [*tutorial_data* folder of this tutorial](../tutorial_data).
 
 2. In the Postman sidebar, click **Task 2 - Upload Source File to OSS > PUT Upload the File**. The request loads.
 
-   Note the use of `ContentUploadSignedURL` in URI.
+   Note the use of `ContentUploadSignedURL` as the URI.
 
 3. Click the **Body** tab.
 
 4. Click **Select File** and select the file *box.ipt*, which you downloaded in step 1.
 
-   ![Select file button](../images/task2_upload_file_2.png "Select file button")
+   ![Select file button](../images/t1_task2_upload_file_2.png "Select file button")
    
 5. Click **Send** to upload the file.
 
```

Comment:
Might be good to add an introductory paragraph for Finalize Upload. For example:

Although you uploaded the source file in one go, it is possible to split a file into chunks and upload the file one chunk at a time. Once all the chunks are uploaded you must inform OSS that the upload operation is complete. Even though ypu uploaded the file in one go, you must inform OSS that the upload is done. To finalize the upload: